docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -0,0 +1,68 @@
+---
+---
+= Clustered Cache
+ifndef::sourcedir[]
+include::common.adoc[]
+endif::sourcedir[]
+
+ifdef::notBuildingForSite[]
+include::menu.adoc[]
+endif::notBuildingForSite[]
+
+== Introduction
+
+As with the standalone cache, in order to use the clustered EhCache you will need to configure Clustered cache manager and the cache.
+This is currently in the active development mode and the complete feature to be out very soon, stay tuned!
+
+=== Clustered cache manager
+
+In addition to the steps performed for creating standalone cache manager, clustered cache manager creation requires you to provide the clustering
+service configuration:
+
+[source,java,indent=0]
+----
+include::{sourcedir}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/GettingStarted.java[tag=clusteredCacheManagerExample]
+----
+
+<1> Static method `org.ehcache.config.builders.CacheManagerBuilder.newCacheManagerBuilder` that returns a new `org.ehcache.config.builders.CacheManagerBuilder`
+    instance;
+<2> Use the ClusteringServiceConfigurationBuilder's static method `.cluster(URI)` for connecting to the clustered cache manager at URI specified and return the
```

Comment:
Isn't this the wrong term, I thought we wanted to avoid calling it the "clustered cache manager"?